actions/transactions/update.go

```diff
@@ -21,19 +21,25 @@ import (
 // @Router		/v1/transaction [patch]
 // @Security	x-auth-xpub
```

Comment:
Add @Deprecated comment, and info in description about what endpoint should be used instead.

Reply:
Fixed.